README.md

```diff
@@ -190,6 +257,38 @@ pymilo_client.toggle_mode(PymiloClient.Mode.DELEGATE)
 result = pymilo_client.predict(x_test)
 ```
 
+#### PymiloClient
+
+The `PymiloClient` class facilitates working with remote PyMilo servers.
+
+| **Parameter** | **Type** | **Description** |
+| ------------- | -------- | --------------- |
+| model | Any | The local ML model to wrap around |
+| mode | Mode | Operating mode (LOCAL or DELEGATE) |
+| compressor | Compression | Compression method for communications |
+| server_url | str | URL of the PyMilo server |
+| communication_protocol | CommunicationProtocol | Communication protocol to use |
+
+| **Mode** | **Description** |
+| -------- | --------------- |
+| LOCAL | Execute operations on the local model |
+| DELEGATE | Delegate operations to the remote server |
```

Comment:
Turn this into a text description, the format used here is different from other place and it can be confusing.